data/core/commands/core.lua

```diff
@@ -86,6 +86,15 @@ command.add(nil, {
   end,
 
   ["core:open-file"] = function()
+    local node = core.root_view:get_active_node_default()
+    local view_idx = node:get_view_idx(core.active_view)
+    if view_idx ~= nil then
+      local view = node.views[view_idx]
+      if view.doc ~= nil and view.doc.filename ~= nil then
+        local directory, filename = view.doc.filename:match("(.*)[/\\](.+)$")
+        core.command_view:set_text(directory .. PATHSEP)
+      end
+    end
     core.command_view:enter("Open File", function(text)
```

Comment:
The function `CommandView.enter` can take a third argument called `suggest`. May be we can use it to add the suggested text.

Reply:
Ok! I've added the directory as the top suggestion in the case where there is no text entered, otherwise, it acts as normal.